.github/workflows/lint-test.yml

```diff
@@ -0,0 +1,71 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+    branches: [ main ]
+  workflow_dispatch: {}
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
```

Comment:
@cdoyle-temple  I would like to have two separate jobs for lint and test.  That way if there are rubocop errors, it fails right away before running the tests.  I want us to be consistent between projects, so as close as you can make this to the isilon-tracker the better.  Do we need the complication of docker compose for TU Press? I'm going to think more about that over lunch.  But I do want lint to be a separate job.